dogs-initial/src/main/resources/templates/dogs.html

```diff
@@ -0,0 +1,33 @@
+<html xmlns:th="http://thymeleaf.org">
+<head>
+    <meta charset="UTF-8" />
+    <title>Display Dog Data</title>
+</head>
+<body>
+<table border="1">
+    <tr>
+        <th>Breed</th>
+        <th>Weight</th>
+        <th>Apartment Suitable</th>
+    </tr>
+    <tr th:each="dog : ${dogList1}">
+        <td th:text="${dog.breed}">Breed</td>
+        <td th:text="${dog.weight}">Weight</td>
+        <td th:text="${dog.apartmentSuitable}">Weight</td>
+    </tr>
+</table>
+<table border="1">
+    <tr>
+        <th>Breed</th>
+        <th>Weight</th>
+        <th>Apartment Suitable</th>
+    </tr>
+    <tr th:each="dog : ${dogList2}">
+        <td th:text="${dog.breed}">Breed</td>
+        <td th:text="${dog.weight}">Weight</td>
+        <td th:text="${dog.apartmentSuitable}">Weight</td>
+    </tr>
+</table>
+<strong><a href="/">Click here to go back</a></strong>
+</body>
+</html>
```

Comment:
Why throw in a link to the root URL here? As the application is right now, it will actually just end up causing a `handlerNotFoundException`.